man/oxide-msg.1

```diff
@@ -1,5 +1,19 @@
-.\" Automatically generated by Pandoc 2.9.2.1
+.\" Automatically generated by Pandoc 2.19.2
```

Comment:
Why are these changes not done in the src-file?

Reply:
maybe this happened automatically